src/components/ec-table-sort/ec-table-sort.spec.js

```diff
@@ -0,0 +1,68 @@
+import { mount } from '@vue/test-utils';
+import { withMockedConsole } from '../../../tests/utils/console';
+import * as SortDirection from '../../enums/sort-direction';
+import EcTableSort from './ec-table-sort.vue';
+
+describe('EcTableSort', () => {
+  function mountEcTableSort(props, mountOpts) {
+    return mount(EcTableSort, {
+      propsData: { ...props },
+      ...mountOpts,
+    });
+  }
+
+  it('should render as expected', () => {
+    const wrapper = mountEcTableSort();
+    expect(wrapper.element).toMatchSnapshot();
+  });
+
+  describe(':props', () => {
+    it('should render with no direction', () => {
+      const wrapper = mountEcTableSort({ direction: null });
+      expect(wrapper.element).toMatchSnapshot();
+    });
+
+    it('should render with direction set to empty string', () => {
+      const wrapper = mountEcTableSort({ direction: '' });
+      expect(wrapper.element).toMatchSnapshot();
+    });
+
+    it('should render with direction set to ASC', () => {
+      const wrapper = mountEcTableSort({ direction: SortDirection.ASC });
+      expect(wrapper.element).toMatchSnapshot();
+    });
+
+    it('should render with direction set to DESC', () => {
```

Comment:
good example where duplication is better than abstraction (having both specs in an array) :+1: